index.js

```diff
@@ -1 +1,94 @@
-console.log('Hello World!');
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const chai = require('chai');
+
+const should = chai.should();
+
+/**
+ * Validates the structure of the "proof" property on a digital document.
+ *
+ * @param {object} options - Options to use.
+ * @param {object} options.data - A digital document with "proof" property on
+ *   it.
+ * @param {string} [options.vendorName] - The name of the vendor who issued the
```

Comment:
It seems like this should be required, otherwise the test cell `columnId` will be undefined?